src/App.tsx

```diff
@@ -23,21 +23,20 @@ var todos = (): Promise<{id: string; title: string;}[]> => new Promise((res) =>
 });
 
 function App() {
-  const [state, setState] = React.useState<{ id: string; title: string }[]>([]);
+  const [todoList, setTodoList] = React.useState<{ id: string; title: string }[]>([]);
 
   React.useEffect(() => {
     (async () => {
-      var awaitedTodos = await todos();
-      for (var i = 0; i < awaitedTodos.length; i++) {
```

Comment:
If you iterate every item, the component will re-render each time when the state is changed. Set all data at once to avoid unnecessary re-render of the component.